way_to_home/way_to_home/settings.py

```diff
@@ -55,7 +55,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'middlewares.login_required.LoginRequiredMiddleware',
+    #'middlewares.login_required.LoginRequiredMiddleware',
```

Comment:
Why did you comment this line?

Reply:
Your code must be working with middlewares